.github/workflows/pgtest.yml

```
@@ -7,10 +7,13 @@ jobs:
   pgtap:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ '14', '15', '16' ]
```

Could you rename it something like: 
```suggestion
        pg-version: [ '14', '15', '16' ]
```